05/app.js

```diff
@@ -1,53 +1,76 @@
 const stats = {
-    paragraphs: {
-        'p1': 0,
-    },
-    links: {
-        '/dolor.html': 0,
-    }
+  paragraphs: {
+    p1: 0,
+  },
+  links: {
+    "/dolor.html": 0,
+  },
 };
 
 /* tutaj umieść swój kod */
 
+const pElements = document.querySelectorAll("p");
+const aElement = document.querySelector("a");
+let value = 0;
+let valueLink = 0;
 
-/* nie modyfikuj kodu poniżej, ale przeanalizuj go */
+const clickCounter = function (e) {
+  e.preventDefault();
```

Comment:
Proponuję wyświetlić sobie w konsoli informacje co pokazuje `e.target` oraz `e.currentTarget` w momencie kliknięcia na `<p>` oraz `<a>`. Wtedy lepiej zrozumiesz działanie propagacja eventów.
Potem w zależności od typu elementu `e.target.tagName === 'P'` możesz wykonać odpowiednie działania.
Daj mi znać na Slacku czy taka podpowiedź będzie wystarczająca ;)

Reply:
Dzięki Mateusz! Mega dużo dała mi ta wskazówka, bo faktycznie nie byłem chyba ostatecznie świadomy tego jakie możliwości daje sam `e.target` a jakie `e.currentTarget`. Dzięki!! 
Zrobiłem tutaj licznik tych, które są już w obiekcie. Czy to jest wystarczające?